topics/admin/tutorials/tool-management/tutorial.md

```diff
@@ -125,50 +146,51 @@ There are additionally two ways to install tools, we'll show both:
 >
 {: .hands_on}
 
-This provides an easy way to do one-off installation of tools, but is less convenient if you want to install many tools. For that, you can install from a yaml file:
+This provides an easy way to do a one-off installation of a tool, but is not very convenient if you want to install many tools.
+For that, you can install from a YAML file:
 
 > ### {% icon hands_on %} Hands-on: Installing tools from a tool list
 >
-> 1. (optional) Use the `tail` command to watch the installation proceed
```

Comment:
@martenson I suppose you meant to watch the Galaxy logs here.

Reply:
looks like it